BasicSample/Assets/ARAnchor/Scripts/AnchorsSample.cs

```diff
@@ -38,10 +38,9 @@ public void ToggleAirTapToCreateEnabled()
         private XRAnchorStore m_anchorStore = null;
         private Dictionary<TrackableId, string> m_incomingPersistedAnchors = new Dictionary<TrackableId, string>();
 
-        async void Start()
+        protected async void OnEnable()
         {
-            m_arAnchorManager = GetComponent<ARAnchorManager>();
-            if (!m_arAnchorManager.enabled || m_arAnchorManager.subsystem == null)
+            if (!TryGetComponent(out m_arAnchorManager) || !m_arAnchorManager.enabled || m_arAnchorManager.subsystem == null)
             {
                 Debug.Log($"ARAnchorManager not enabled or available; sample anchor functionality will not be enabled.");
                 return;
```

Comment:
we should make the similar changes as our TestProject recent changes, keep the "anchor manager" in this start function, and move the "anchorStore" to a separate async function that may last longer.

Reply:
I've synced this file with the internal test project now. It was already pretty similar to my changes, luckily!